test/integration/smoke/test_iso.py

```diff
@@ -320,8 +320,8 @@ def test_02_edit_iso(self):
             "Check display text of updated ISO"
         )
         self.assertEqual(
-            iso_response.bootable,
-            self.services["bootable"],
+            str(iso_response.bootable).lower(),
```

Comment:
@borisstoyanov I am very worried the we are hiding issues this way. A test that has been passing for about five years suddenly fails and then we fix the test? I realize this *might* not be a issue with the code/system. Do we have any clue as to when and why it started failing?

Reply:
ping @rhtyd @PaulAngus @nvazquez We've spend some time at this with @DaanHoogland and figured it might have been a library or something else that has updated causing this. Please shout if you have any idea.

Reply:
Latest changes in list response, marvin test file and test_data.py does not seem to be causing this...

Reply:
I don't have any idea, where/how it may change. LGTM, I think we can merge this and revisit later if needed.